homeassistant/components/vera/__init__.py

```diff
@@ -56,34 +57,62 @@
     """Set up for Vera controllers."""
     config = base_config.get(DOMAIN)
 
-    if config is None:
+    entries = hass.config_entries.async_entries(DOMAIN)
+    import_entries = [
+        entry
+        for entry in entries
+        if entry.data.get(CONF_SOURCE) == config_entries.SOURCE_IMPORT
+    ]
+
+    if not config:
+        for entry in import_entries:
+            _LOGGER.debug(
+                "Removing existing import config for %s",
+                entry.data.get(CONF_CONTROLLER),
+            )
+            await hass.config_entries.async_remove(entry.entry_id)
```

Comment:
It's not good to remove the existing entry since users will loose their customizations.